### PR TITLE
Five files changed.

### PR DESCRIPTION
**Pull request by loserlulin9**
This is my first PR, so maybe have some problems.
I have fixed some bugs in these docs(NO.219-227 in "docs working group清单").
Please check them and tell me how can I make them better.
Much thx!!!

### CODE REVIEW

**Comment by loserlulin9** [comment]
Doc-NEW自动审查报错：**You must have one TPM (jzhang533/ZhangJun or momozi1996/MoYan or dingjiaweiww/DingJiaWei or Ligoml/LiMengLiu or sunzhongkai588/SunZhongKai) approval for the api change! docs/api/paddle for the management reason of API interface and API document.** 请问该如何解决？

**Comment by sunzhongkai588** [comment]
> Doc-NEW自动审查报错：**You must have one TPM (jzhang533/ZhangJun or momozi1996/MoYan or dingjiaweiww/DingJiaWei or Ligoml/LiMengLiu or sunzhongkai588/SunZhongKai) approval for the api change! docs/api/paddle for the management reason of API interface and API document.** 请问该如何解决？

经过我的review后即可解决

File: docs/api/paddle/logit_cn.rst [hunk 1]
```diff
@@ -5,12 +5,12 @@ logit
...
 .. py:function:: paddle.logit(x, eps=None, name=None)
 
-实现了 logit 层。若 eps 为默认值 None，并且 ``x`` < 0 或者 ``x`` > 1，该函数将返回 NaN，OP 的计算公式如下：
+实现了 logit 层。若 eps 为默认值 None，并且 ``x`` < 0 或者 ``x`` > 1，该函数将返回 NaN，计算公式如下：
 
 .. math::
     logit(x) = ln(\frac{x}{1-x})
 
-其中，:math:`x`` 为输入的 Tensor，且和 eps 有着如下关系：
+其中，:math: ``x`` 为输入的 Tensor，且和 eps 有着如下关系：
```
Comment (sunzhongkai588) [inline docs/api/paddle/logit_cn.rst]: `:math: `后面不跟空格，这里可以把 `:math: ` 删除了，只留  ``x`` ，不影响阅读体验
image

File: docs/api/paddle/mode_cn.rst
Comment (sunzhongkai588) [inline docs/api/paddle/mode_cn.rst:1]: 该文档的问题在于

- 代码输出和注释给的结果不一致，需要去paddle仓库下源码改动 https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/tensor/search.py#L549  可以在paddle repo 下提pr修复

image

- 少了[源代码]，应该是 `.. py:function:: paddle.mode(x, axis=-1, keepdim=False,  name=None):` 最后多了个冒号，删除即可


image

**Review by sunzhongkai588 — COMMENTED** [review]
@loserlulin9 几处细节需要优化～

File: docs/api/paddle/mode_cn.rst
Comment (loserlulin9) [reply docs/api/paddle/mode_cn.rst:1]: 收到，我去优化下。

Comment (loserlulin9) [reply docs/api/paddle/mode_cn.rst:1]: `` mode() ``函数代码示例修改见PR：https://github.com/PaddlePaddle/Paddle/pull/53082

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Comment by loserlulin9** [comment]
所有检查均已通过，请问该PR能被合并了吗？

**Pull request merged**